blog.araya.dev/src/posts/2021-08-12-speculation-rules-prefetch.md

```diff
@@ -67,7 +67,7 @@ Chrome および Google では、Google が提供しているサービスから
 この Google が提供するサービス での例は[Chromium Blog](https://blog.chromium.org/2020/12/continuing-our-journey-to-bring-instant.html)でが紹介されたものだが、記事執筆時点で筆者が Google Search で検証した限りは Private Prefetch Proxy 経由での prefetch や prerender は発見できなかった。
 将来的になにかアナウンスがあるかもしれない。
 
-### Opt-in / Opt-out
+## Opt-in / Opt-out
 
 ### Opt-out
```

Comment:
I thought it was meant to be  <h2 /> tag here because of the context and the broken styling. (if my intuition was wrong, feel free to revert this change 🙏 )

![Screen Shot 2021-08-12 at 1 20 18](https://user-images.githubusercontent.com/42742053/129066423-ef0b9313-34fc-41f2-b217-46ad36be2d34.png)